rosa_architecture/rosa-understanding.adoc

```diff
@@ -5,7 +5,7 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-understanding
 toc::[]
 
-Learn about {product-title} (ROSA), interacting with ROSA using {cluster-manager-first} and command-line interface (CLI) tools, consumption experience, and integration with Amazon Web Services (AWS) services.
+Learn about {product-title} (ROSA), interacting with ROSA using {cluster-manager-first} and the {product-title} (ROSA) CLI (`rosa`) tools, consumption experience, and integration with Amazon Web Services (AWS) services.
```

Comment:
No need for two expansions in one sentence; just use "ROSA CLI (`rosa`)"

Reply:
Maybe we need to revoke change here? The existing line looks fine to me.